src/Microsoft.Win32.SystemEvents/src/Configurations.props

```diff
@@ -1,14 +1,8 @@
 ﻿<Project>
   <PropertyGroup>
-    <PackageConfigurations>
+    <BuildConfigurations>
       netstandard;
-      netcoreapp2.0-Windows_NT;
       netcoreapp-Windows_NT;
```

Comment:
We shouldn't be putting the version less TargetGroups in packages as it sets a bad-precedent for unbounded growth of the package.

Reply:
That makes sense. I think we have quite a lot places where we have version less TargetGroups in packages. Let's follow-up on that in another PR.

Reply:
https://github.com/dotnet/corefx/issues/36577